.github/scripts/clone_engine_repo

```diff
@@ -78,6 +78,10 @@ else
             BRANCH_OPTION=''
         fi
     fi
+    if [ "$BRANCH" = "jira-babel-3801" ]; then
```

Comment:
Same here, we don't need this temp change.

Reply:
Done!